stratum/hal/lib/tdi/tdi_constants.h

```diff
@@ -70,6 +70,29 @@ constexpr uint64 kMaxPriority = (1u << 24) - 1;
 
 constexpr absl::Duration kDefaultSyncTimeout = absl::Seconds(1);
 
+// ES2k specific const used in PacketModMeter
```

Comment:
Maybe add the statement ->
These numbers are present in the enum prefix for Externs in message P4IDs present in the es2k specific p4info.proto file.

Reply:
Meaning that the Es2k prefix will be dropped when the p4info.proto changes become part of the official release, and we will adopt the change then?

I'm okay with that. It's a way of marking these definitions as provisional.

Reply:
If by "official release" you mean p4lang, these prefixes won't go there. These prefixes are under `OTHER_EXTERNS_START` section which is supposed to be target specific. That's why Tofino, DPDK and ES2K have overlaps

Reply:
Added comment